documentation/src/docs/index.md

```diff
@@ -49,6 +51,12 @@ You can find a list of frequently asked questions [here](faq.md).
 
 Pravega can be installed locally or in a distributed environment. The installation and deployment of Pravega is covered in the [Running Pravega](deployment/deployment.md) guide.
 
+## Pravega Security, Role based access control and TLS
+Pravega supports encryption of all communication channels and pluggable role based access control.
+Find more details here:
+1. [TLS](security/pravega-security-encryption.md)
+2. [Authorization, authentication and RBAC](security/pravega-security-authorization-authentication.md)
+
 ## Support
```

Comment:
I think a `security@pravega.io` address would be a much better way to communicate with us on security issues. We should set it up.